.github/workflows/windows.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       id: openvino_download
       with:
         platform: 'windows'
-        commit_packages_to_provide: 'wheels'
+        commit_packages_to_provide: wheels,openvino_js_package.zip
```

Comment:
@mryzhov is `.zip` required here?

Reply:
Yes, extensions is required
it was discussed [here](https://github.com/openvinotoolkit/openvino.genai/pull/1667#discussion_r1956304522)